src/commcare_cloud/terraform/modules/logshipping/main.tf

```
@@ -8,16 +8,28 @@ locals {
 
 resource "aws_s3_bucket" "log_bucket" {
   bucket = local.log_bucket_name
-  acl = "private"
+  #https://github.com/hashicorp/terraform-provider-aws/issues/23888
+  lifecycle {
+    ignore_changes = [
+    acl,
+    server_side_encryption_configuration
+  ]
+}
```

I think these lines should be indented
```suggestion
      acl,
      server_side_encryption_configuration
    ]
  }
```